.baseline/checkstyle/checkstyle.xml

```diff
@@ -56,6 +56,11 @@
         <property name="message"
                   value="Prefer using Maps.newHashMap instead."/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="new ConcurrentHashMap&lt;&gt;\(.*\)"/>
+        <property name="message"
```

Comment:
This looks good to me but does it need to be two lines?

Reply:
fixed